enterprise/backend/src/metabase_enterprise/sandbox/api/error_page.mustache

```diff
@@ -8,7 +8,7 @@
       <div style="width:1000px;">
         <img src="../app/assets/img/disconnect.svg">
 
-        <h2>It looks like we weren't able to log you in.</h2>
+        <h2>It looks like we weren't able to log you {{logDirection}}.</h2>
```

Comment:
Why does this file live at `metabase_enterprise/sandbox/api/` 🤔 

No need to change it necessary since it was there before, but seems odd. Don't see how this is related to sandboxing.